<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactDialog</class>
 <widget class="QDialog" name="ContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contact</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>First Name:</string>
       </property>
       <property name="buddy">
        <cstring>firstNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="firstNameEdit">
       <property name="placeholderText">
        <string>Enter first name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>Last Name:</string>
       </property>
       <property name="buddy">
        <cstring>lastNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lastNameEdit">
       <property name="placeholderText">
        <string>Enter last name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="phoneLabel">
       <property name="text">
        <string>Phone:</string>
       </property>
       <property name="buddy">
        <cstring>phoneEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="phoneEdit">
       <property name="placeholderText">
        <string>Enter phone number</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Email:</string>
       </property>
       <property name="buddy">
        <cstring>emailEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="emailEdit">
       <property name="placeholderText">
        <string>Enter email address</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="notesLabel">
     <property name="text">
      <string>Notes:</string>
     </property>
     <property name="buddy">
      <cstring>notesEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="notesEdit">
     <property name="placeholderText">
      <string>Enter additional notes</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>firstNameEdit</tabstop>
  <tabstop>lastNameEdit</tabstop>
  <tabstop>phoneEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>notesEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
